app/views/services/autoform/index.html.erb

```diff
@@ -4,12 +4,12 @@
 
 <div class="container autoform">
   <h1><strong>Autoform</strong></h1>
-  <p class="lead">Automatické vypĺňanie kontaktných a fakturačných údajov o právnických osobách. <br><em>Vždy aktuálne údaje, rýchle a bez chýb.</em></p>
+  <p class="lead">Automatické vypĺňanie kontaktných a fakturačných údajov o právnických osobách a SZČO. <br><em>Vždy aktuálne údaje, rýchle a bez chýb.</em></p>
```

Comment:
Dal by som skor `o právnických osobách a živnostníkoch` nez skratku.

Reply:
Ano, suhlasim, znie to lepsie.